lib/markdown-it/index.js

```diff
@@ -34,7 +34,7 @@ markdownIt.normalizeLink = require('./normalizeLink');
 
 // fix table style
 markdownIt.renderer.rules.table_open = (tokens, idx) => {
-  return '<table class="markbind-table table">';
+  return '<table class="markbind-table table-bordered table-striped table">';
```

Comment:
Why did `table` become the last class?

Reply:
My bad. Should it be ` markbind-table table table-bordered table-striped` instead? Based on alphabetical order.

Reply:
Yes. Sometimes special classes like `markbind-table` can be excluded from the "alphabetical order" convention.